### ISSUE TITLE
nhentai website is blocking rapid api calls

### ISSUE THREAD

**Issue by claron-akatsuki** [RicterZ/nhentai#46]
![image](https://user-images.githubusercontent.com/46862558/52987375-cd130e80-3435-11e9-952c-136e814e4fc2.png)
This activates on your ip address. I think it can be circumvented by clicking on the approve button, but I do not know if they have any further protection against scraping.

**Comment by RicterZ** [comment]
I got it, I'll consider some actions to against / bypass the protection.

**Comment by WhosMyName** [comment]
For a quick Workaround:
just import sleep and add a 2 second sleep between parsing iterations in nhentai/command.py 
```python3
from time import sleep

#Line 46
    if doujinshi_ids:
        for id_ in doujinshi_ids:
            doujinshi_info = doujinshi_parser(id_)
            doujinshi_list.append(Doujinshi(**doujinshi_info))
->         sleep(2)
```
@RicterZ If ya need help, just holla ;-)

**Comment by RicterZ** [comment]
I released version 0.3.1, you can specify a option `--login` to log into nhentai.net, and keep the session during nhentai runtime.  It maybe can byass the limit I think... If you meet any problems please submit a issue.
